src/test/java/com/google/sps/PostTest.java

```diff
@@ -0,0 +1,230 @@
+package com.google.sps;
+
+import com.google.sps.data.Post;
+import javax.servlet.http.HttpServletRequest;
+import org.junit.Assert;
+import org.junit.runners.JUnit4;
+import org.junit.runner.RunWith;
+import org.junit.Test;
+import static org.mockito.Mockito.*;
```

Comment:
Would recommend to not use the `*` catch all because of reasons listed [here](https://medium.com/@aayushsharda/why-is-it-better-to-avoid-in-import-statements-in-java-ff9ceaca8aeb)!

Reply:
Oh cool, thanks!